File: java/ControlFlow.java
// control flow examples

public class ControlFlow {
	
	public static void main(String [] args) {
		
		int a = 1;
		
		// ternary operator
		boolean b = (a == 1) ? true : false;

		// if-statement
		if(a < 0) {
			System.out.println("minor");
		}
		else if(a == 0) {
			System.out.println("zero");
		}
		else {
			System.out.println("major");
		}
		
		// switch-statement
    int i=2;
        
		switch(i){
			case 0:
				System.out.println("i is zero");
				break;
			case 1:
				System.out.println("i is one");
				break;
			case 2:
				System.out.println("i is two");
				break;
			case 3:
				System.out.println("i is three");
				break;
			default:
				System.out.println("i is something other");
        }
    } 		
}

File: languages/c/control-flow.c
#include <stdio.h>

int main(int argc, char **argv) {
	int a = 0;
	
	// if-statement
	if (a < 0) { 
		printf("minor");
	} else if(a == 0) {
		printf("zero");
	} else {
		printf("major")
	}
	
	// switch-statement
	char b = 'x';
	switch (someChar) {
		case 'a': printf('a'); break;
		case 'x': printf('x'); break;
		case 'y':
		case 'z': printf('y or z'); break;
		default: printf('default');
	}
	
	return 0;
}